src/migrations/1732280885581-create-test-data.ts

```diff
@@ -0,0 +1,173 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CreateTestData1732280885581 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        queryRunner.query(`
+        INSERT INTO public.association
+        (id, "name", logo, description, telegram, phone, email, membership_fee)
```

Comment:
Inserting IDs manually breaks the migration in case there is already data in the database. Don't put an ID and the database will generate ids on its own.